hbase-common/pom.xml

```diff
@@ -212,9 +221,24 @@
       <scope>test</scope>
     </dependency>
     <dependency>
-      <!--For Hadoop-->
+      <groupId>org.slf4j</groupId>
+      <artifactId>jcl-over-slf4j</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>jul-to-slf4j</artifactId>
```

Comment:
Whats jcl and jul?

Reply:
Some of our code uses these?

Reply:
java commons-logging and java.util.logging, at least when I tried to ban all the dependencies to commons-logging it was failed, as there are some libraries we use depend on commons-logging, so redirecting the jcl and jul to slf4j could help us to get more log messages from depended libraries, no harm.